### DISCUSSION TITLE
CLI ncurses output: show conf file used, and URL

### DISCUSSION THREAD

**Discussion by sanderjo** [nzbgetcom/nzbget#289] [Ideas]
When you start nzbget, it just says

```
INFO    nzbget 24.1 server-mode
```
And that's it.
To me it's unclear:

- which conf file is used
- what url to use to access nzbget

I had to use `strace` and `netstat -tulpen` to find that out ... :-(

Proposal: write both info in the CLI output

```
INFO    nzbget 24.1 server-mode
INFO    using /etc/nzbget.conf
INFO    access on http://127.0.0.1:6789/
```

The current view:

![image](https://github.com/nzbgetcom/nzbget/assets/1273502/18d72ed6-46d1-4680-b1e9-1dac54984b8c)

**Comment by luckedea** [comment]
Good idea, useful suggestion.

**Comment by FDrebin** [comment]
I agree with this, other items that utilize NZBGet do this already, Sonarr, Radarr, and NZBHydra2 . This would be a great addition for new and existing users.

**Comment by dnzbk** [comment]
Done and already available in the current test build with the new “STATUS” tab - welcome to test.

**Comment by sanderjo** [comment]
Got the test version from https://github.com/nzbgetcom/nzbget/releases, tested it, and ... it works! Beautiful. Thank you!

![image](https://github.com/user-attachments/assets/39250e9d-fee9-4c10-922b-2e07036850b0)



```
 Files for downloading                                                                                  Uptime 00:00:37
Ready to receive nzb-job
 Messages
INFO    nzbget 24.2-testing-20240801 server-mode
INFO    using /home/sander/Downloads/nzbget/nzbget.conf
INFO    nzbget runs on 0.0.0.0:6789


```
